docs/diagrams/ImportCsvSequenceDiagram.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "command:ImportCsvCommand" as ImportCsvCommand LOGIC_COLOR
+end box
+
+box Storage STORAGE_COLOR_T1
+participant ":CsvManager" as CsvManager MODEL_COLOR
+end box
+
+[-> LogicManager : execute(importcsv)
```

Comment:
`importcsv` should be shown as a string.

Reply:
thanks for the suggestion, implemented in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/203/commits/3193b02116aeddfa74a438d365ea5575d9acfb0b